storage/src/main/java/org/apache/uniffle/storage/handler/impl/AbstractClientReadHandler.java

```diff
@@ -38,10 +43,32 @@ public void close() {
   }
 
   @Override
-  public void updateConsumedBlockInfo(BufferSegment bs) {
+  public void updateConsumedBlockInfo(BufferSegment bs, boolean skipped) {
+    if (bs == null) {
+      return;
+    }
+    updateBlockMetric(readHandlerMetric, bs, skipped);
   }
 
   @Override
   public void logConsumedBlockInfo() {
+    LOG.info("Client read [" + readHandlerMetric.getReadBlockNum() + " blocks,"
+        + " bytes:" +  readHandlerMetric.getReadLength() + " uncompressed bytes:"
+        + readHandlerMetric.getReadUncompressLength()
+        + "], skipped[" + readHandlerMetric.getSkippedReadBlockNum() + " blocks,"
+        + " bytes:" +  readHandlerMetric.getSkippedReadLength() + " uncompressed bytes:"
+        + readHandlerMetric.getSkippedReadUncompressLength() + "]");
+  }
+
+  protected void updateBlockMetric(ClientReadHandlerMetric metric, BufferSegment bs, boolean skipped) {
```

Comment:
Could we give `skipped` a better name?

Reply:
discarded？

Reply:
> I can't get the meaning of this parameter. Do it mean that this BufferSegment will be skipped if this value is true?

There are two kinds of block meitrcs in this pr, `Consumed` and `Skipped`. `Consumed` means the block has not been read before and `Skipped` means block has been read before or the block is damaged.

Reply:
`updateSkippedMetrics` may be better.

Reply:
> `updateSkippedMetrics` may be better.

Add a new method?

Reply:
> > `updateSkippedMetrics` may be better.
> 
> Add a new method?

It's ok to rename this parameter

Reply:
Maybe `isSkippedMetrics` may be better. Because `updateSkippedMetrics` is like a method name. [emo]

Reply:
> It's ok to rename this parameter

I don't get your point. Rename `skipped` or add a new method? `ClientReadHandler#updateConsumedBlockInfo` use `skipped` as a parameter too. Should we add a new method like`updateSkippedBlockInfo(BufferSegment bs)`?

Reply:
> Maybe `isSkippedMetrics` may be better. Because `updateSkippedMetrics` is like a method name. [emo]

So, rename `skipped` to  `isSkippedMetrics`?

Reply:
Yes.